### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by Callisto13**
woops

### CODE REVIEW

File: tests/integration/integration_test.go [hunk 1]
```diff
@@ -699,10 +699,10 @@ status: {}
 
 			By("creating the artifacts")
 			Expect(files).To(ContainElements(
-				"artifacts/bitnami-nginx/ConfigMap.yaml",
-				"profile-installation.yaml",
-				"artifacts/bitnami-nginx/HelmRelease.yaml",
-				"artifacts/bitnami-nginx/HelmRepository.yaml",
+				"nginx/artifacts/bitnami-nginx/ConfigMap.yaml",
```
Comment (Skarlso) [inline tests/integration/integration_test.go:702]: didn't I already do this?

Comment (Callisto13) [reply tests/integration/integration_test.go:702]: 🤷‍♀️  says my branch is up to date.

it was failing for other reasons tho, this was just extra

Comment (Skarlso) [reply tests/integration/integration_test.go:702]: https://github.com/weaveworks/pctl/commit/08f02c34024343fa3a9fc8770981af566399e7c0#diff-e4ede531ad8643dc7af3ffb5ddf644bdfa5c2f8a45430f580f0fa687ffab06efR702

What the heck. :D

Comment (Skarlso) [reply tests/integration/integration_test.go:702]: wonky diff url... but this is the PR: https://github.com/weaveworks/pctl/pull/142/files#

Comment (Skarlso) [reply tests/integration/integration_test.go:702]: Oh!!!!!!!! It was your merge conflict resolution which reverted my fix. :D

Comment (Skarlso) [reply tests/integration/integration_test.go:702]: https://github.com/weaveworks/pctl/commit/a139fa55ef6501a159a75438eb7e1397e5bd0151#diff-e4ede531ad8643dc7af3ffb5ddf644bdfa5c2f8a45430f580f0fa687ffab06efL702

**Review by aclevername — APPROVED** [review]

**Pull request merged**